client/Dockerfile-5.12

```
@@ -145,10 +140,28 @@ RUN if [ "$BUILD_QT" = "1" ] ; then \
 ENV QT_TARBALL qt-bin-${VER_QT}-openssl-${VER_OPENSSL}-linux-x86_64-${VER_QT_DATE}.tar.xz
 
 RUN if [ "$BUILD_QT" != "1" ] ; then echo Download precompiled Qt. && \
-      apt install -y xz-utils && \
       wget https://download.nextcloud.com/desktop/development/qt/${QT_TARBALL} && \
       tar -xvf ${QT_TARBALL} && \
       rm ${QT_TARBALL} \
     ; fi
 
 ###########################################################################
+
+ENV QTDIR ${QT_ROOT}
+ENV PATH ${QT_ROOT}/bin:${PATH}
+ENV LD_LIBRARY_PATH ${QT_ROOT}/lib/x86_64-linux-gnu:${QT_ROOT}/lib:/usr/local/lib:${LD_LIBRARY_PATH}
+ENV PKG_CONFIG_PATH ${QT_ROOT}/lib/pkgconfig:${PKG_CONFIG_PATH}
+
+# Install QtKeychain
+RUN cd /tmp && \
+    git clone https://github.com/frankosterfeld/qtkeychain.git && \
+    cd qtkeychain && \
+    git checkout v0.10.0 && \
```

This does a shallow clone of exactly the commit `v0.10.0` and should save time and traffic. (we do this also on our CI to improve performance)
```suggestion
    git clone https://github.com/frankosterfeld/qtkeychain.git --depth 1 -b v0.10.0 && \
```